pkgs/applications/editors/emacs/elisp-packages/manual-packages/color-theme-solarized/default.nix

```diff
@@ -1,26 +1,41 @@
 { lib
-, trivialBuild
-, fetchFromGitHub
 , color-theme
+, fetchFromGitHub
+, melpaBuild
+, writeText
 }:
 
-trivialBuild {
-  pname = "color-theme-solarized";
-  version = "0.pre+unstable=2017-10-24";
+let
+  argset = {
+    pname = "color-theme-solarized";
+    ename = "color-theme-solarized";
+    version = "20230209.837";
 
-  src = fetchFromGitHub {
-    owner = "sellout";
-    repo = "emacs-color-theme-solarized";
-    rev = "f3ca8902ea056fb8e46cb09f09c96294e31cd4ee";
-    hash = "sha256-oxX0lo6sxotEiR3nPrKPE9H01HKB3ohB/p8eEHFTp5k=";
-  };
+    src = fetchFromGitHub {
+      owner = "sellout";
+      repo = "emacs-color-theme-solarized";
+      rev = "b186e5d62d0b83cbf5cf38f7eb7a199dea9a3ee3";
+      hash = "sha256-7E8r56dzfD06tsQEnqU5mWSbwz9x9QPbzken2J/fhlg=";
+    };
+
+    packageRequires = [
+      color-theme
+    ];
```

Comment:
Not needed since https://github.com/sellout/emacs-color-theme-solarized/commit/30dfe48d811fbbe7180bda3b98bf10e4037b6061.

```suggestion
```